pkgs/development/tools/language-servers/nil/default.nix

```
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nil";
-  version = "2023-03-11";
+  version = "2023-04-03";
 
   src = fetchFromGitHub {
     owner = "oxalica";
     repo = pname;
     rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
It seems to be encouraged to use `refs/tags/` for tags recently.

---

this shouldn't be needed unless you deliberately create a branch named `2023-04-03`